### ISSUE TITLE
Issue with changing theme in dw_edit_chart

### ISSUE THREAD

**Issue by michaelgoodier** [munichrocker/DatawRappr#59]
Really useful package for a current project i'm having to work on at work - thanks!

For some reason I was unable to change the name of the datawrapper theme using the following dw_edit_chart(). 
(x here being the name of a dw_chart-object)

```
dw_edit_chart(x, title="Example title",
                type="d3-bars-grouped",
                theme="custom_theme")
```

I managed to add it using a custom httr::Patch() 

```
call_body <- list(title="Example title",
                    type="d3-bars-grouped",
                    theme = dw_theme,
                    metadata = list(describe = list(`source-name` = "Global Data")))    
  
  httr::PATCH(url = paste0('https://api.datawrapper.de/v3/charts/',x$id), httr::add_headers(Authorization = paste0('Bearer ',api_key)), body = call_body, encode = "json",.DATAWRAPPR_UA)
```
I'm not sure why - maybe it is to do with the order of things in the call body as as theme isn't a default option in dw_edit_chart it gets added to the end? Thought I'd flag anyway!

**Comment by munichrocker** [comment]
Seems to be an issue with `rlist::list.append()`. It's appending a nested item to `call_body`, instead of a vector element (`theme = "xyz"`).

I am testing a fix it with base `append()`

**Comment by munichrocker** [comment]
This issue should be fixed in V1.2 (https://github.com/munichrocker/DatawRappr/commit/599bfac27c4716b674b82946c584c2810b324ce0)
